gdi/get-data-in/application/java/configuration/advanced-java-otel-configuration.rst

```diff
@@ -252,7 +252,7 @@ The following settings control metrics collection for the Java agent:
    * - ``SPLUNK_METRICS_ENDPOINT``
      - The OTel collector metrics endpoint. Default is ``http://localhost:9943``. |br| |br| System property: ``splunk.metrics.endpoint``
    * - ``SPLUNK_METRICS_EXPORT_INTERVAL``
-     - Interval between pushing metrics. You can define duration in the form ``<number><unit>``, where the unit can be ``ms``, ``s``, ``m``, ``h``, or ``d``. The default interval is ``30s``. If you enter a number but not a unit, the default unit is assumed to be ``ms``. |br| |br| System property: ``splunk.metrics.export.interval``
+     - Deprecated. Use ``OTEL_METRIC_EXPORT_INTERVAL`` instead. See https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#periodic-exporting-metricreader
```

Comment:
I hate this and I am not smart enough to figure out where to hide urls in this `.rst` thing.

Reply:
Thanks! Don't worry about the link, I'm going to edit your branch directly. FWIW, the `:new-page:` directive we use for external links is not part of Sphinx or rST, it's a macro we developed for opening links in a new tab.